.gitignore

```diff
@@ -15,9 +15,14 @@ __pycache__/
 .idea/
 .venv/
 
+functionaltest-report.xml
 smoketest-report.xml
-env
+.env
 .dir-locals.el
 *.pyc
-
+*-report.xml
+*.key
+*.pem
+sandbox/coverage
+sandbox/.nyc_output
```

Comment:
What generates these? we don;t have much in the sandbox yet

Reply:
have removed as not used